central/views/imagecve/view_impl.go

```diff
@@ -197,7 +197,7 @@ func withSelectCVECoreResponseQuery(q *v1.Query, cveIDsToFilter []string, option
 	cloned := q.Clone()
 	if len(cveIDsToFilter) > 0 {
 		cloned = search.ConjunctionQuery(cloned, search.NewQueryBuilder().AddDocIDs(cveIDsToFilter...).ProtoQuery())
-		cloned.Pagination = q.GetPagination()
+		cloned.Pagination = &v1.QueryPagination{SortOptions: q.GetPagination().GetSortOptions()}
```

Comment:
We should check for q.GetPagination() to be nil before we clobber, and use if not nil right (otherwise we clobber it)

Reply:
Good point. I added nil checks. And yes this is the only path that is pre-paged. For non paginated queries, `cveIDsToFilter` would be empty.